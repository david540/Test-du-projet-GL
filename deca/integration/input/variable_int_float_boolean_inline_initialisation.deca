{
    int x = 3;
    float y = 5.6;
    boolean b = false;
    println(x, y);
}
