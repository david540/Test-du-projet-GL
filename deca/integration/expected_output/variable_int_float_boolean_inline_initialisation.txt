35.60000e+00
